src/backend/common/models/history.py

```diff
@@ -0,0 +1,9 @@
+from typing import List, TypedDict
+
+from backend.common.models.award import Award
+from backend.common.models.event import Event
+
+
+class History(TypedDict):
+    events: List[Event]
+    awards: List[Award]
```

Comment:
is the intent to add some sort of `stats` field in the future here as well?

Reply:
Likely